packages/trader/src/Modules/Account/Sections/Security/LoginHistory/login-history.jsx

```diff
@@ -0,0 +1,161 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { DesktopWrapper, MobileWrapper } from '@deriv/components';
+import { convertDateFormat } from '@deriv/shared/utils/date';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import DataTable from 'App/Components/Elements/DataTable';
+import Loading from '../../../../../templates/app/components/loading.jsx';
+import LoadErrorMessage from '../../ErrorMessages/LoadErrorMessage';
+
+const Td = ({ title, text, className }) => (
+    <td>
+        <h3 className='cell-title'>{title}</h3>
+        <span className={classNames('cell-value', className)}>{text}</span>
+    </td>
+);
+
+const LoginHistoryListView = ({ fields, login_history }) => (
```

Comment:
could our current `<Table>` component from components be used instead?

Reply:
It's a special one with respect to styles and will be a mess if I use `<Table>`.

Reply:
ok 👍